rudder-inventory-ldap/SPECS/rudder-inventory-ldap.spec

```diff
@@ -287,7 +287,12 @@ if [ -n "${BACKUP_LDIF}" ]; then
 		# Stop OpenLDAP - use forcestop to avoid the init script failing
 		# when trying to do the backup with bad libdb versions
 		echo -n "INFO: Stopping rudder-slapd..."
-		service rudder-slapd forcestop >/dev/null 2>&1
+      %if 0%{?rhel} < 7
```

Comment:
Hmm, we have the same line in lot of ther packages (reports / relay ... ) I think it's using the first one